docs/faqGeneral.md

```diff
@@ -8,35 +8,43 @@ fabric8 uses the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0
 
+#### What is Kubernetes?
+
+[Kubernetes](http://kubernetes.io) provides a [Docker](http://docker.io/) based _Container As A Service_ (orchestrates docker containers)
+
+#### How does Fabric8 relate to other open source projects?
+
+Fabric8 provides upstream innovation on concepts and features, once validated, make their way into other open source projects such as [OpenShift Origin](https://github.com/openshift/origin), [Kubernetes](http://kubernetes.io) and [Jenkins](https://jenkins.io/) (as examples). Fabric8 contributors work in the various other open source communities to align a unified developer experience.
+
 #### What value does fabric8 add over OpenShift?
 
-* [Kubernetes](http://kubernetes.io) provides a [Docker](http://docker.io/) based _Container As A Service_ (orchestrates docker containers)
 * [OpenShift V3](https://github.com/openshift/origin) extends Kubernetes to support a full _Platform As A Service_
-  * hosting source code in git repositories
-  * performing builds and hosting private docker images
-  * supporting the git-push style model of kicking off new builds
+  * isolation of source code [builds within containers](https://docs.openshift.org/latest/dev_guide/builds.html)
+  * providing a [simple console](https://docs.openshift.org/latest/getting_started/developers_console.html) so you can create, view, modify and remove all of your Kubernetes and OpenShift resources
+  * ease of use for [creating and deploy from source or docker formatted image](https://docs.openshift.org/latest/getting_started/developers_cli.html#developers-cli-creating-an-application)
+  * supporting the [git-push style model](https://docs.openshift.org/latest/dev_guide/builds.html#build-triggers) of kicking off new builds
+  * keep base container layers current with [automatic rebuilds on image changes](https://docs.openshift.org/latest/dev_guide/builds.html#build-triggers)
+  * integrated application [metrics](https://docs.openshift.org/latest/install_config/cluster_metrics.html) and [logging](https://docs.openshift.org/latest/install_config/aggregate_logging.html)
+  * large set of [builder images](https://docs.openshift.org/latest/using_images/s2i_images/index.html) and [quick starts](https://github.com/openshift/origin/tree/master/examples)
+  * flexible [deployment models](https://docs.openshift.org/latest/dev_guide/deployments.html): rolling, blue/green and [load balancing for A/B testing](https://docs.openshift.org/latest/dev_guide/routes.html#routes-load-balancing-for-AB-testing)
 * Fabric8 is focused on:
-  * providing a great [Kubernetes Console](console.html) based on [hawtio](http://hawt.io/) so you can view all of the kubernetes resources, understand whats going on and see the big picture or zoom inside inside individual containers and link to logs and metrics
```

Comment:
but aside from technology used, there isn't anything added on top of OpenShift's console based on this description. So it may need to go in a different section/sub-section on what is different about the consoles and why.